src/qt/guiutil.cpp

```diff
@@ -1670,15 +1706,22 @@ QFont::Weight supportedWeightFromIndex(int nIndex)
     return vecWeights[nIndex];
 }
 
-int supportedWeightToIndex(QFont::Weight weight)
+bool supportedWeightToIndex(QFont::Weight weight, int& nIndex)
```

Comment:
Why doing it like that instead of simply returning `-1` on failure?

Reply:
Good question :smile: see c948413e2cb708ac186a0dbee29f2d0b22f35fe8